templates/index.html

```diff
@@ -135,23 +135,28 @@ <h2 class="col-xs-12 text-center">Naši partneři</h2>
             </a>
         </div>
         <div class="partner col-xs-6 col-sm-4">
-            <a href="http://pyvec.org/" target="_blank" rel="noopener">
-                <img class="partner-logo img-responsive center-block" src="{{ pathto('_static/img/logo/pyvec.svg', 1) }}" />
+            <a href="https://www.nic.cz/" target="_blank" rel="noopener">
+                <img class="partner-logo img-responsive center-block" src="{{ pathto('_static/img/logo/cznic.svg', 1) }}" />
             </a>
         </div>
         <div class="partner col-xs-6 col-sm-4">
-            <a href="http://www.experis.cz/" target="_blank" rel="noopener">
-                <img class="partner-logo img-responsive center-block" src="{{ pathto('_static/img/logo/manpowergroup.svg', 1) }}" />
+            <a href="https://www.tieto.cz/" target="_blank" rel="noopener">
+                <img class="partner-logo img-responsive center-block" src="{{ pathto('_static/img/logo/tieto.svg', 1) }}" />
             </a>
         </div>
         <div class="partner col-xs-6 col-sm-4">
-            <a href="https://apiary.io/" target="_blank" rel="noopener">
-                <img class="partner-logo img-responsive center-block" src="{{ pathto('_static/img/logo/apiary.svg', 1) }}" />
+            <a href="https://msdit.cz/" target="_blank" rel="noopener">
```

Comment:
ja si uprimne myslim, ze uz ani jako nejsou sponzori dost dlouhou dobu. za me ma prednost apiary, ktere podporuje dlouhodobe a pravidelne. cc @zuzejk vis o tom, ze by msd jeste nejak podporovalo PyLadies?

Reply:
a chapu, ze to neni v popisu prace tohohle PR, ale kdyby to zuzejk potvrdil, mohlo by se to rovnou i updatovat ❤️

Reply:
To pořadí je **skoro** jen podle toho, aby to vizuálně vyšlo, neměl sem čas to předělat aby bylo jedno, kde který logo je a nebyly na sebe nalepený atd.

Reply:
Samozřejmě MSD se vyhodit může, pokud to není aktuální.

Reply:
čau, MSD teď sice pravda nevyužíváme, ale občas se nějak přes někoho připomenou, že tam máme možnost uspořádat akce